public/JS/hero-carousel.js

```
@@ -0,0 +1,27 @@
+// Image Carousel
+let currentSlide = 0;
+const images = document.querySelectorAll('.carousel-image');
+const totalSlides = images.length;
+let autoAdvanceInterval;
+
+function showSlide(index) {
+    images.forEach(img => img.classList.remove('active'));
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)